### ISSUE TITLE
Colorize output of cp-deploy.sh

### ISSUE THREAD

**Issue by luigimolinaro** [IBM/cloud-pak-deployer#323]
There is a way to colorize output of the script.
It's not easy to debug if no color is showed

I'm speaking about this for example : (should be yellow) 

TASK [cpd-cli-download : Get current version number of cpd-cli] ****************
Sunday 12 February 2023  09:37:53 +0000 (0:00:00.167)       0:00:23.319 *******

**Comment by fketelaars** [comment]
@luigimolinaro I agree that colored output would be better and looked into this. We have chosen to run the deployer as a daemon of podman/docker and tail the logs. This causes the colors to be removed unfortunately. If you want to show the colorized output, you could do the following:
* Set the CONFIG_DIR and STATUS_DIR environment variables
* `./cp-deploy.sh env cmd`
* `/cloud-pak-deployer/cp-deploy.sh env apply`

This causes the command to be run "interactively" and you will see output in color.

**Comment by luigimolinaro** [comment]
I will try do add this feature. I get this point

**Comment by luigimolinaro** [comment]
"I found a way to do this using ccze, but we do not want to require its installation. However, we have the option to do so if necessary."

cp-deploy.sh : 

Add function to check if ccze exist : 
```bash
#Check if CCZE exist
function find_ccze() {
  if command -v ccze >/dev/null 2>&1; then
    CCZE_COMMAND="true"
  fi
}
find_ccze
```

and rewrite the funciont run_env_logs()
```bash
# Show the logs of the currently running env process
run_env_logs() {
  if [[ "${ACTIVE_CONTAINER_ID}" != "" ]]; then
    while ${CPD_CONTAINER_ENGINE} ps --no-trunc 2>/dev/null | grep -q ${ACTIVE_CONTAINER_ID}; do
      if [ ${CCZE_COMMAND} == true ]; then
        ${CPD_CONTAINER_ENGINE} logs -f --tail 10 ${ACTIVE_CONTAINER_ID} | ccze -m ansi
      else
        ${CPD_CONTAINER_ENGINE} logs -f --tail 10 ${ACTIVE_CONTAINER_ID}
        if [ $? -ne 0 ]; then
          break
        fi
        sleep 0.5
      fi
    done
  else
    ${CPD_CONTAINER_ENGINE} logs ${CURRENT_CONTAINER_ID}
  fi
}
```

**Comment by luigimolinaro** [comment]
Color is now available ! 
I'm so proud of this gift for the "deployer" :) 

<img width="895" height="542" alt="Image" src="https://github.com/user-attachments/assets/0132d2dc-6aa9-488c-b110-83575d374180" />
